src/Microsoft.Azure.SignalR/HubHost/ServiceLifetimeManager.cs

```diff
@@ -15,6 +15,10 @@ namespace Microsoft.Azure.SignalR
 {
     internal class ServiceLifetimeManager<THub> : HubLifetimeManager<THub> where THub : Hub
     {
+        private const string MarkerNotConfiguredError =
+            "'UseAzureSignalR(...)' not called after calling 'AddAzureSignalR(...)'." +
```

Comment:
> UseAzureSignalR(...)' not called after calling 'AddAzureSignalR(...). Please always use 'AddAzureSignalR(...)' and 'UseAzureSignalR(...)' altogether.

Change this to.

AddAzureSignalR(...) was called without matching call to IApplicationBuilder.UseAzureSignalR(...).

We should also point to an FWLink that points to the docs on how to configure the service.

Reply:
Sure.
We don't have docs specifically for configuring the service. There is supposed to be a FAQ page in docs.microsoft.com. We can put this issue in there. But it is not ready yet. I will ask what is the progress.